apps/web/src/constants/twitter.ts

```diff
@@ -0,0 +1,3 @@
+import { baseUrl } from '~/utils/baseUrl';
+
+export const TWITTER_AUTH_URL = `https://twitter.com/i/oauth2/authorize?response_type=code&client_id=T0RhNDBVSWdVVGh2ZzBOdHJobHA6MTpjaQ&redirect_uri=${baseUrl}/auth/twitter&scope=tweet.read%20users.read%20follows.read%20follows.write%20offline.access&state=state&code_challenge=challenge&code_challenge_method=plain`;
```

Comment:
Should we put the `client_id` in the env file?

Reply:
yeah that would be sick

Reply:
let's make the entire thing an ENV variable!

Reply:
Moved it to environmental variables!